code_schemes/s01e14.json

```diff
@@ -201,6 +201,70 @@
         "StringValue": "rumour_stigma_misinfo_misinfo_on_status",
         "VisibleInCoda": true
       },
+      {
+        "CodeID": "code-57cdd2ee",
+        "CodeType": "Normal",
+        "DisplayText": "provide support",
+        "NumericValue": 22,
+        "StringValue": "provide_support",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-b4bdf141",
+        "CodeType": "Normal",
+        "DisplayText": "encouragement",
+        "NumericValue": 23,
+        "StringValue": "encouragement",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-53128241",
+        "CodeType": "Normal",
+        "DisplayText": "practice farming/cultivation",
+        "NumericValue": 24,
+        "StringValue": "practice_farming_cultivation",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-629ae1b9",
+        "CodeType": "Normal",
+        "DisplayText": "livestock treatment/veterinary service",
+        "NumericValue": 25,
+        "StringValue": "livestock_treatment_veterinary_service",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-998d0a32",
+        "CodeType": "Normal",
+        "DisplayText": "consume livestock products",
+        "NumericValue": 29,
+        "StringValue": "consume_livestock_products",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-22cda656",
+        "CodeType": "Normal",
+        "DisplayText": "protects the environment",
+        "NumericValue": 26,
+        "StringValue": "protects_the_environment",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-2dfa5986",
+        "CodeType": "Normal",
+        "DisplayText": "livestock;the buttress of the society",
```

Comment:
```suggestion
        "DisplayText": "livestock; the buttress of the society",
```
or
```suggestion
        "DisplayText": "livestock, the buttress of the society",
```

Reply:
This has since been pushed to coda and fully labeled. Will leave as is and consider the changes in the future.